src/hooks/useLoadFromEndpoint.ts

```diff
@@ -21,7 +21,7 @@ export const loadFromEndpoint = async <T, P>(
     const response = await request(params)
     setData(response)
     setError(null)
-  } catch (e) {
+  } catch (e: any) {
```

Comment:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-4.html#defaulting-to-the-unknown-type-in-catch-variables---useunknownincatchvariables